views/about.twig

```diff
@@ -5,171 +5,182 @@
-
-            <p>PHPDorset provides online and offline opportunities to:
+    <div class="wrapper clearfix document">
+        <h3>Quick Jump</h3>
+        <ul>
+            <li><a href="#code-of-conduct">Code of Conduct</a>
                 <ul>
-                    <li>discuss PHP and related topics</li>
-                    <li>socialise with friendly people who are interested in PHP</li>
-                    <li>find or fill PHP related jobs</li>
+                    <li><a href="#incident-handling">Incident Handling</a></li>
                 </ul>
-            </p>
-
-            <p>We want PHPDorset to be a diverse and inclusive community, united by an interest in PHP, and so we want anyone who is interested in PHP to be able to participate.</p>
-
-            <ul>
-                <li><a href="#code-of-conduct">Code of Conduct</a>
-                    <ul>
-                        <li><a href="#incident-handling">Incident Handling</a></li>
-                    </ul>
-                </li>
-                <li><a href="#volunteers">Volunteer Team</a></li>
-                <li><a href="#meetings">Meetings</a>
-                    <ul>
-                        <li><a href="#talks">Talks</a>
-                            <ul>
-                                <li><a href="#scheduling">Scheduling</a></li>
-                                <li><a href="#subject-matter">Subject matter</a></li>
-                                <li><a href="#timing">Timing</a></li>
-                                <li><a href="#after-talks">After the talks</a></li>
-                            </ul>
-                        </li>
-                    </ul>
-                </li>
-                <li><a href="#sponsorship">Sponsorship</a></li>
-                <li><a href="#about-this">About this document</a></li>
-            </ul>
-
-            <h2><a name="code-of-conduct">Code of Conduct</a></h2>
-
-            <p>It is of primary importance that everyone who wants to participate feels safe and welcome. When participating in PHPDorset, either online or offline, at St Ambrose or at the pub afterwards, we expect you to:
+            </li>
+            <li><a href="#volunteers">Volunteer Team</a></li>
+            <li><a href="#meetings">Meetings</a>
                 <ul>
-                    <li>behave respectfully towards other people — they’re real humans, just like you</li>
-                    <li>be considerate of people’s time and attention</li>
-                    <li>contribute positively and constructively to discussions (online or in-person)</li>
+                    <li><a href="#talks">Talks</a>
+                        <ul>
+                            <li><a href="#scheduling">Scheduling</a></li>
+                            <li><a href="#subject-matter">Subject matter</a></li>
+                            <li><a href="#timing">Timing</a></li>
+                            <li><a href="#after-talks">After the talks</a></li>
```

Comment:
After talks in the sub-menu here, but made into it's own section in the content.  Should it either be included in the Talks section or the menu item put to the top level to maintain consistency in the document structure